CHANGELOG.md

```diff
@@ -1,3 +1,168 @@
+
+*deprecated* method: `-mutate:` in `MDCAppBarTextColorAccessibilityMutator`
+
+### Dialogs
+
+#### MDCAlertController
+
+*new* property: `buttonInkColor` in `MDCAlertController`
+
+#### MDCAlertControllerView
+
+*new* property: `buttonInkColor` in `MDCAlertControllerView`
+
+### FlexibleHeader
+
+#### MDCFlexibleHeaderView()
+
+*moved* category: `MDCFlexibleHeaderView()`
+
+*modified* property: `contentView` in `MDCFlexibleHeaderView()`
+
+## Component changes
+
+### MaskedTransition
+
+#### Changes
+
+* [[List] Fix Swift imports (#4828)](https://github.com/material-components/material-components-ios/commit/b917bf24d60029d76bf29114a2c1526793dfe6b1) (Robert Moore)
+
+### FeatureHighlight
+
+#### Changes
+
+* [Fix crash when backgrounding and re-opening app. (#4796)](https://github.com/material-components/material-components-ios/commit/44bed97140da63cee922b70bcfd152ebfc984c13) (Andrew Overton)
+
+### AppBar
+
+#### Changes
+
+* [Make migration links be actual links (#4808)](https://github.com/material-components/material-components-ios/commit/4efe02d07a4d49a474acb3e4495fa61073550d57) (featherless)
+* [Mark MDCAppBarTextColorAccessibilityMutator as deprecated. (#4807)](https://github.com/material-components/material-components-ios/commit/8c394dc2728eb9d107fbed705932acb4fed8aa80) (featherless)
+
+### Buttons
+
+#### Changes
+
+* [fix the text overlapping safeArea issue on](https://github.com/material-components/material-components-ios/commit/2444107e30b81d921b650208373fa4c6c4239ae1) (Wenyu Zhang)
+
+### TextFields
+
+#### Changes
+
+* [Trigger layout pass to update borderView.borderLayer (#4815)](https://github.com/material-components/material-components-ios/commit/13bff72b262ffb521432a80665db3727fc4dcbe5) (Andrew Overton)
+
+### Chips
+
+#### Changes
+
+* [Example respects safe area (#4856)](https://github.com/material-components/material-components-ios/commit/6e6a6bf827908b7b78cb776c038b23d159d2996d) (Joe Aguilar)
+
+### Cards
+
+#### Changes
+
+* [Fix Swift imports (#4826)](https://github.com/material-components/material-components-ios/commit/ae8fcb4e473a2beb9a55d1c3f99442c718244792) (Robert Moore)
+* [Fix example IBOutlet (#4827)](https://github.com/material-components/material-components-ios/commit/bf099ab14fcf0d9222534cbf67f37ef7250c9a42) (Robert Moore)
+
+### BottomAppBar
+
+#### Changes
+
+* [Fix typo in the readme (#4869)](https://github.com/material-components/material-components-ios/commit/215011b3866b47724553cde55b5239d057dc5455) (Brad Mallow)
+
+### ShadowLayer
+
+#### Changes
+
+* [Revert "Setting the corner radius should recalculate the shadowPath (#4804)"](https://github.com/material-components/material-components-ios/commit/bd8a8153bc318365e2e76c42a7f6e16e76267ac0) (Ian Gordon)
+
+### List
+
+#### Changes
+
+* [[MaskedTransition] Fix Swift imports (#4828)](https://github.com/material-components/material-components-ios/commit/b917bf24d60029d76bf29114a2c1526793dfe6b1) (Robert Moore)
```

Comment:
Same here - remove the "[MaskedTransition]" bit.

Reply:
Note that changes that affect multiple components shouldn’t be prefixed with the individual component names for this reason.

Reply:
How should we prefix changes to multiple components? I thought @material-automation expects to see the subject line prefixed with the component names in square braces?